stdlib/messaging/kafka/src/main/ballerina/src/kafka/consumer.bal

```diff
@@ -375,24 +377,24 @@ public type Consumer client object {
         return consumerGetPositionOffset(self, partition, duration);
     }
 
-# Retrieves the set of topics which are currently subscribed by the consumer.
-# ```ballerina
-# string[]|kafka:ConsumerError result = consumer->getSubscription();
-# ```
-#
-# + return - Array of subscribed topics for the consumer if executes successfully or else `kafka:ConsumerError`
+    # Retrieves the set of topics which are currently subscribed by the consumer.
```

Comment:
```suggestion
    # Retrieves the set of topics, which are currently subscribed by the consumer.
```

Reply:
Why aren't these reviewed in API doc hackathon?